proto/prysm/v1alpha1/payload_attestation.proto

```diff
@@ -41,3 +41,9 @@ message PayloadAttestationMessage {
     PayloadAttestationData data 	= 2;
     bytes signature 		= 3 [(ethereum.eth.ext.ssz_size) = "96"];
 }
+
+message IndexedPayloadAttestation {
```

Comment:
`IndexedPayloadAttestation` doesn't have to be a protobuf; it can be a native Go type. We only define something as a protobuf when we need SSZ functionality, like hash tree roots, but `IndexedPayloadAttestation` doesn't require consensus.